package.json

```diff
@@ -62,6 +62,7 @@
     "tough-cookie": ">=4.1.3"
   },
   "dependencies": {
+    "@btitterington_org/docx": "^8.5.1",
```

Comment:
How does this differ from https://www.npmjs.com/package/docx?

Reply:
Sorry, only just seen this now. This is a fork that lets us use the json format when patching a document (rather than manually having to build up a section of the document using individual docx components). We can create an official Oak fork of the docx repo, it was just a bit quicker to do it this way initially.

Reply:
**Note to self**: We've moved from using the docx package to our own wrapper, remove this prior to merge